build.gradle

```diff
@@ -32,17 +32,17 @@ subprojects {
     }
 
     compileJava {
-        sourceCompatibility = JavaVersion.VERSION_1_7
-        targetCompatibility = JavaVersion.VERSION_1_7
+        sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Could you please separate the Java version upgrade into separate PR?

Reply:
done